t/socket.t

```diff
@@ -55,7 +57,7 @@ ok(defined $server_socket->{_inet_socket});
 $select = IO::Select->new();
 ok($server_socket->add_handles($select));
 $server_socket->add_handles($select);
-ok($select->count() == 2);
+ok($select->count() eq 2);
```

Comment:
This should actually be the numeric comparison *==*; the *eq* operator is for string comparison.